webpack/assets/javascripts/react_app/components/common/CopyToClipboard/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { OverlayTrigger, Tooltip, Button } from 'patternfly-react';
+import { CopyIcon } from '@patternfly/react-icons';
+import { translate as __ } from '../../../common/I18n';
+
+const CopyToClipboard = ({ valueToCopy }) => {
+  const inputRef = useRef(null);
+  const tooltip = <Tooltip id="copy-btn-tooltip">{__('Copied!')}</Tooltip>;
+
+  const handleCopy = () => {
+    inputRef.current.select();
+    document.execCommand('copy');
+  };
+
+  return (
+    <div className="input-group">
+      <input
+        className="form-control"
+        id="input_to_copy"
```

Comment:
could there be more than one of these components in a page? or - do we actually need the id?